cmd/cli/list/list.go

```diff
@@ -263,3 +270,35 @@ func summarizeJob(j *model.JobWithInfo, OL *ListOptions) (table.Row, error) {
 
 	return row, nil
 }
+
+func shortenTime(outputWide bool, t time.Time) string { //nolint:unused // Useful function, holding here
+	if outputWide {
+		return t.Format("06-01-02-15:04:05")
+	}
+
+	return t.Format("15:04:05")
+}
+
+var DefaultShortenStringLength = 20
+
+func shortenString(outputWide bool, st string) string {
+	if outputWide {
+		return st
+	}
+
+	if len(st) < DefaultShortenStringLength {
+		return st
+	}
+
+	return st[:20] + "..."
+}
+
+func ShortID(outputWide bool, id string) string {
```

Comment:
do you need to expose this method outside of list package?

Reply:
Yes its used in list_test.go which is a separate package named `list_test`.

Reply:
it may be set to `true` by a user flag, so will leave in place for now.